src/components/posts/PostsPage.js

```diff
@@ -4,68 +4,74 @@ import PostForm from './PostForm';
 import client from '../../utils/client';
 import './style.css';
 import { Box, Stack } from '@mui/material';
+import Avatar from '@mui/material/Avatar'
 import Header from '../Header/Header';
 import dateTimetoRelativeTime from './helperfunctions';
 import { Link } from 'react-router-dom';
 
 const PostsPage = ({ role }) => {
-  const [post, setPost] = useState({ content: '' });
-  const [postResponse, setPostResponse] = useState('');
-  const [posts, setPosts] = useState([]);
+	const [post, setPost] = useState({ content: '' });
+	const [postResponse, setPostResponse] = useState('');
+	const [posts, setPosts] = useState([]);
 
-  useEffect(() => {
-    client.get('/posts').then((res) => setPosts(res.data.data.posts));
-  }, []);
+	useEffect(() => {
+		client.get('/posts').then((res) => setPosts(res.data.data.posts));
+	}, []);
 
-  const createPost = async (event) => {
-    event.preventDefault();
-    client
-      .post('/post', post)
-      .then((res) => {
-        setPostResponse(res.data);
-        setPosts((posts) => [res.data.data.post, ...posts]);
-      })
-      .catch((data) => {});
-    setPost(() => ({ content: '' }));
-  };
+	const createPost = async (event) => {
```

Comment:
Check your formatting settings - usually don't want to have these types of changess in a PR, makes it hard to see the actual code change,